src/Game.js

```diff
@@ -13,9 +13,9 @@ class Game {
   }
 
   start() {
```

Comment:
Nice job renaming that parameter to attempt to shorten the line to bring it closer to 80